### PR TITLE
blog/2023: A couple of fixes for the reddit challenge post

### PR DESCRIPTION
**Pull request by markhpc**

### CODE REVIEW

File: src/en/news/blog/2023/reddit-is-10k-IOPs-achieable-with-NVMes/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ tags:
...
...again for another blog post!  Recently, a user on the ceph subreddit [asked](https://www.reddit.com/r/ceph/comments/14ztb22/is_10k_iops_achievable_with_nvmes/) whether Ceph could deliver 10K IOPS in a combined random read/write FIO workload from one client.  The setup consists of 6 nodes with 2 4TB FireCuda NVMe drives each.  They wanted to know if anyone would mind benchmarking a similar setup and report the results.  Here at [Clyso](https://clyso.com) we are actively working on improving the Ceph code to achieve higher performance.  We have our own tests and configurations for evaluating our changes to the code, but it just so happens that one of the places we do our work (the upstream ceph community performance lab!) appears to be a good match for testing this user's request.  We decided to sit down for a couple of hours and give it a try.  [u/DividedbyPi](https://www.reddit.com/user/DividedbyPi/), one of our friends over at [45drives.com](https://www.45drives.com/), wrote that they are also going to give it a shot and report the results on their youtube channel in the coming weeks.  We figure this could be a fun way to get results from multiple vendors.  Let's see what happens!
```
Comment (anthonyeleven) [inline src/en/news/blog/2023/reddit-is-10k-IOPs-achieable-with-NVMes/index.md:17]: Ceph subreddit
YouTube

**Review by anthonyeleven — APPROVED** [review]
caps

**Comment by dvanders** [comment]
@markhpc one more "They didn't mentioned" --> "They didn't mention"

**Comment by dvanders** [comment]
@markhpc ping latest comment

**Pull request merged**